### DISCUSSION TITLE
Remove custom clone property

### DISCUSSION THREAD

**Discussion by anbraten** [woodpecker-ci/woodpecker#2474] [Request a Feature]
Instead of using the `clone` property in the config its possible to write as supported since https://github.com/woodpecker-ci/woodpecker/pull/2163:

```diff
- clone:
-   git:
-     image: woodpeckerci/plugin-git
-    settings:
-      partial: false
-      depth: 50

steps:
  build:
   image: debian
   commands:
      - aws help
```

```diff
+ skip_clone: true

steps:
+ clone:
+    image: woodpeckerci/plugin-git
+    settings:
+      partial: false
+      depth: 50
  build:
   image: debian
   commands:
      - aws help
```

**Comment by xoxys** [comment]
I would like to keep the possibility to set the clone image globally as server config property.

**Reply by qwerty287** [reply]
I don't think this was the idea to remove it, this discussion is only about the custom clone step in workflow YAMLs.

**Comment by 6543** [comment]
This will break services that use files of the repo
